800.renames-and-merges/a.yaml

```diff
@@ -88,6 +88,7 @@
 - { setname: agg,                      name: [antigraingeometry,libagg] }
 - { setname: agg,                      name: asciinema-agg }
 - { setname: ags,                      name: aylurs-gtk-shell }
+- { setname: aioblescan,               name: "python:aioblescan" }
```

Comment:
Since it's announced as a library, it should probably be merged the other way around.